scene/gui/menu_button.cpp

```diff
@@ -100,6 +114,7 @@ void MenuButton::set_disable_shortcuts(bool p_disabled) {
 
 MenuButton::MenuButton() {
 
+	switch_on_hover = false;
```

Comment:
Also, I think it's better to be true by default. As all other software are.

Reply:
Making it true by default would be chaotic, as it would also trigger icon-text/icon-only MenuButtons.

Reply:
And I think all users will expect that MenuButton works with mouse hover like all other software do when creating your own UI with godot.

Reply:
Another example:
![captura de tela de 2018-08-04 11-28-32](https://user-images.githubusercontent.com/1387165/43677520-956a1fbc-97d9-11e8-8ac1-10da5355a6da.png)
I think this button should open as well.

When I made my PR I forgot about these buttons with icons and texts.

Reply:
Maybe we could let buttons with only icons open automatically too in same cases. like "snap" and "bones" in 2D viewport.

Reply:
Here is another button menu that was left behind:
![captura de tela de 2018-08-04 11-44-46](https://user-images.githubusercontent.com/1387165/43677651-d4f2b8cc-97db-11e8-8fac-3be3c589cf66.png)
"View" and "Animation" has the same parent and should open on hover.

Reply:
These menus are from different "sections", generally separated with VSeparators, or far apart from each other. It doesn't make sense for them to switch on hover like they're part of the same group.